backend/app/utils.py

```
@@ -66,7 +66,9 @@
                 "full_name": id_info.get("name"),
             }
         except ValueError as e:
-            raise HTTPException(status_code=400, detail=f"Token verification failed: {str(e)}")
+            raise HTTPException(
+                status_code=400, detail=f"Token verification failed: {str(e)}"
+            )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)